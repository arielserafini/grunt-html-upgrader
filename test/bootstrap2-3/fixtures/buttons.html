<div class="btn-group">
  <a class="btn" href="{{ asset('something') }}">Default</a>
  <a class="btn btn-primary">Primary</a>
</div>
