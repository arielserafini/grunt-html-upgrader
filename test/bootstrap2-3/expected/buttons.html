<div class="btn-group">
  <a class="btn btn-default">Default</a>
  <a class="btn btn-primary">Primary</a>
</div>
